Vagrantfile

```diff
@@ -0,0 +1,11 @@
+Vagrant.configure(2) do |config|
+
+    config.vm.box = "ubuntu/trusty64"
+
+    config.vm.provision :shell, :path => "vagrant/bootstrap.sh"
+
+    config.vm.provider "virtualbox" do |v|
+        v.customize ["modifyvm", :id, "--memory", 1024]
+    end
+
```

Comment:
Could we add a synced folder to export rulerz into the VM? (even if commented by default)

Something like this:

```
Vagrant.configure(2) do |config|
    # …

    # uncomment these lines to share rulerz's repository with the VM
    #config.vm.network "private_network", type: "dhcp"                                                                                                                                                            
    #config.vm.synced_folder ".", "/home/vagrant/workspace", type: "nfs"
end